app/views/shared/_text_tooltip.erb

```diff
@@ -0,0 +1,5 @@
+<div role="tooltip" data-tooltip-target="tooltip" class="tooltip bg-gray-700 text-xs p-1 rounded">
```

Comment:
```suggestion
<div role="tooltip" data-tooltip-target="tooltip" class="tooltip bg-gray-700 text-sm px-1.5 py-1 rounded-md">
```

Just a few tweaks based on official specs.  My bad for not including Figma file!

Reply:
Puzzle in this step, Comprehensive clear  but on moving steps ahead may hire wire,

Reply:
My dad never input the figma files